### DISCUSSION TITLE
I can't Deploy Template Project

### DISCUSSION THREAD

**Discussion by SenseiSea** [vercel/community#4234] [Help]
### Summary

I want to select a template and deploy it, but I get an error before the build starts.

### Steps to Reproduce

I created an account from vercel and I connected a github account of mine but when I try to deploy a template project I get this problem every time;

An unexpected error occurred. Our team has already been notified and are working to resolve the issue, please try again shortly.

**Comment by amyegan** [comment]
Sorry you're having trouble deploying a new project, @SenseiSea. If you share a link to the template, I can look into it for you 🙏

**Reply by SenseiSea** [reply]
Thanks for your reply,
https://vercel.com/templates/next.js/nextjs-portfolio-pageview-counter

**Reply by amyegan** [reply]
Thanks for sharing that. I just deployed a new project from that template and didn't get an error. Do you have Upstash connected and set up correctly for the project?

**Reply by SenseiSea** [reply]
Sorry for the my late reply. Yes, I think I have everything set up correctly. I get the same error when I try to create other templates.
I have recorded a video about this error.
https://streamable.com/f8yl1u
I am recording from my phone but I have the same problem on my computer.
